guppylang/tys/parsing.py

```diff
@@ -82,11 +82,11 @@ def _try_parse_defn(node: AstNode, globals: Globals) -> Definition | None:
     match node:
         case ast.Name(id=x):
             if x not in globals:
-                raise GuppyError("Unknown identifier", node)
+                raise GuppyError(f"Unknown identifier: {x}", node)
```

Comment:
Drive-by: Printing the unknown identifiers here and on line 89

Reply:
See #479. It also makes the printed id show the module name instead of the object repr.